### DISCUSSION TITLE
How does using a smart connect client that log in a smart connect server avoid my notes to go on the web ?

### DISCUSSION THREAD

**Discussion by coulas** [brianpetro/smart-connect#25] [Help & Support]
Hello Brian,
There is a claim that using Smart Connect prevents me from sending my notes to the cloud.
But it does that by using a Smart Connect client and server...
I would appreciate documentation and commitment about how it works.
I guess Smart Connect clients expose embeddings only, and the Smart Connect server is just a router (the GPT assistant knows where the server is, and the server is connected to our client using something (SSE, web socket, ?).

Thanks for the good work on Smart Connect and Connections.

**Comment by coulas** [comment]
PS: maybe emphasing that using chatGPT for embeddings is sending documents to the cloud of OpenAI and that people may look at their own documentation and commitments about confidentiality and data reuse.

**Reply by hittjwX** [reply]
Very good question. In my mind, the only way data stays in my environment is for a local LLM to digest and embed my notes. If anything from the `.smart-connections` folder leaves my environment that's a problem.

I've been able to confirm most publicly available AI is training on some user data. That's because I've been able to add new (and false) ideas to models, then recall them in a different session. 

An example, building out biographical information about a book author where it didn't exist previously. I'll introduce 10 facts, of which 8 are real, one is false but difficult to verify.

Then in another session and user, I'd get at least half the "facts" returned in a query. It's not instant, but the injected false facts show up. Some AI are worse than others. That's the big concern about notes in any form making it to LLM.

Frankly, some of my notes would make AI dumb. Other notes contain confidential or proprietary information. Either way, I don't trust other peoples notes to be accurate. Until I can afford to have an Ollama server isolated on my network, what controls need to be in place to prevent data leaks?
